### PR TITLE
RHDEVDOCS 5960 Errata RN update for GitOps 1.9.4 - 1.9

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

this version only

Issue:

RHDEVDOCS 5960

Link to docs preview:

https://73660--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes#gitops-release-notes-1-9-4_gitops-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

A respin of #73216 with a one-word style correction and correct version destinations - this version is for gitops-docs-1.9. See #73216 for Dev and QE approvals.

### CODE REVIEW

**Review by JoeAldinger — COMMENTED** [review]
A few questions

File: modules/gitops-release-notes-1-9-4.adoc [hunk 1]
```diff
@@ -8,12 +8,31 @@
 
 {gitops-title} 1.9.4 is now available on {OCP} 4.12, 4.13, and 4.14.
 
+[id="errata-updates-1-9-4_{context}"]
+== Errata updates
+
+[id="rhsa-2024-0691-gitops-1-9-4-security-update-advisory_{context}"]
+=== RHSA-2024-0691 - {gitops-title} 1.9.4 security update advisory
+
+Issued: 2024-02-05
```
[the remark this answers is not in the document]
Comment (mramendi) [reply modules/gitops-release-notes-1-9-4.adoc:17]: I have checked with the developers and they insist this is the correct issue date: "This was a re-shipping of erratas so we maintained original dates, when they were initially shipped"

[the remark this answers is not in the document]
Comment (JoeAldinger) [reply modules/gitops-release-notes-1-9-4.adoc:17]: I guess the issue is that the metadata is not going to match the issue date so perhaps there is a middle ground where you could add a line saying that this was re-issued or updated on 03-22

**Pull request merged**